server/svix-server/src/lib.rs

```diff
@@ -47,9 +47,7 @@ pub mod worker;
 
 const CRATE_NAME: &str = env!("CARGO_CRATE_NAME");
 
-lazy_static! {
-    pub static ref SHUTTING_DOWN: AtomicBool = AtomicBool::new(false);
-}
+pub static SHUTTING_DOWN: Lazy<AtomicBool> = Lazy::new(|| AtomicBool::new(false));
```

Comment:
This can just be `pub static SHUTTING_DOWN: AtomicBool = AtomicBool::new(false)`, since that become a `const fn` since this code was originally written.

Reply:
Aha! Good catch